_components/in-page-navigation/accessibility-tests.md

```diff
@@ -0,0 +1,11 @@
+---
+permalink: /components/in-page-navigation/accessibility-tests/
+layout: accessibility-test
+component:
+ name: in page navigation
```

Comment:
Frontmatter is YAML, we do two spaces for consistent indents.

```suggestion
  name: in page navigation
```

Reply:
Resolved in [5b4a912](https://github.com/uswds/uswds-site/pull/2863/commits/5b4a9120e93079ea8d1cc7b6e1c4a847bd6afb73)